modular_newhorizons/master_files/code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -0,0 +1,24 @@
+/singleton/reagent/toxin/potassium_chlorophoride
+	description = "A specific chemical based on Potassium Chloride to stop the heart for surgery. Not safe to eat!"
+
+/singleton/reagent/lexorin
+	description = "Lexorin temporarily stops respiration. Causes tissue damage."
+
+/singleton/reagent/soporific
+	description = "An effective hypnotic used to treat insomnia, can act as a sedative. Lasts three times longer when inhaled."
+
+/singleton/reagent/polysomnine
+	name = "Chloral Hydrate"
+	description = "A powerful sedative. Lasts two times longer when inhaled."
```

Comment:
"Chloral Hydrate is a complex drug which rapidly induces sedation in preparation for surgery. Chloral Hydrate’s sedative effect is fast acting, and sedated individuals wake up with zero amnesia regarding the events leading up to their sedation, however the only downside is how hard the drug is on the liver."